tests/pages/pay-page.js

```diff
@@ -1,16 +1,34 @@
-import { clickable, collection, create, text, triggerable, visitable } from 'ember-cli-page-object';
+import { clickable, collection, create, text, visitable } from 'ember-cli-page-object';
 import AccountDropdown from 'codecrafters-frontend/tests/pages/components/account-dropdown';
 import Header from 'codecrafters-frontend/tests/pages/components/header';
 
 export default create({
   accountDropdown: AccountDropdown,
-  clickOnApplyRegionalDiscountButton: clickable('[data-test-apply-regional-discount-button]'),
+
```

Comment:
Let's stick to convention and not add spaces unless there's a nested block that needs spaces around it (all single-line properties don't need spaces around them)